src/components/Integrations/BigQuery/FreeAccount/FreeAccount.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { BigQueryLogo } from './styles/styles';
+import { useIntl } from 'react-intl';
+import datastudiogif from '../../../../img/google-data-studio.gif';
+import { FormattedMessageMarkdown } from '../../../../i18n/FormattedMessageMarkdown';
+
+interface IFreeAccountProps {}
+
+const FreeAccount: React.FunctionComponent<IFreeAccountProps> = (props) => {
+  const intl = useIntl();
+  const _ = (id: string, values: any) => intl.formatMessage({ id: id }, values);
```

Comment:
If you are using typescript and `values` parameter is optional, I think that you should define that.

I am not sure what is the notation, but probably something like this:

```suggestion
  const _ = (id: string, values?: any) => intl.formatMessage({ id: id }, values);
```

Or

```suggestion
  const _ = (id: string, values: any?) => intl.formatMessage({ id: id }, values);
```

Reply:
Suggest accepted and applied